src/dnum.ts

```diff
@@ -76,18 +76,16 @@ export function from(
 export function setValueDecimals(
   value: Value,
   decimalsDiff: Decimals,
-  options: { round?: boolean } = {},
+  options: { round?: Rounding } = {},
```

Comment:
We should probably use `rounding` rather than `round`, for consistency (e.g. `format()` uses `decimalsRounding`).

Reply:
I have renamed all instances of `round` to `rounding`